opencti-platform/opencti-front/src/components/saved_filters/SavedFilterCreateDialog.tsx

```diff
@@ -0,0 +1,103 @@
+import React, { useState, ChangeEvent } from 'react';
+import DialogTitle from '@mui/material/DialogTitle';
+import DialogContent from '@mui/material/DialogContent';
+import DialogActions from '@mui/material/DialogActions';
+import Button from '@mui/material/Button';
+import Dialog from '@mui/material/Dialog';
+import { useFormatter } from 'src/components/i18n';
+import TextField from '@mui/material/TextField';
+import { graphql } from 'react-relay';
+import { useDataTableContext } from 'src/components/dataGrid/components/DataTableContext';
+import { insertNode } from 'src/utils/store';
+import useApiMutation from '../../utils/hooks/useApiMutation';
+import getSavedFilterScopeFilter from './getSavedFilterScopeFilter';
+
+const savedFilterCreateDialogMutation = graphql`
+  mutation SavedFilterCreateDialogMutation($input: SavedFilterAddInput!) {
+    savedFilterAdd(input: $input) {
+      id
+      name
+      filters
+      scope
+    }
+  }
+`;
+
+type SavedFilterDialogProps = {
+  onClose: () => void;
+  isOpen: boolean;
+};
+
+const SavedFilterCreateDialog = ({ isOpen, onClose }: SavedFilterDialogProps) => {
+  const { t_i18n } = useFormatter();
+
+  const {
+    useDataTablePaginationLocalStorage: {
+      localStorageKey,
+      viewStorage: { filters },
+    },
+  } = useDataTableContext();
+
+  const [filterName, setFilterName] = useState<string>();
+
+  const [commit] = useApiMutation(
+    savedFilterCreateDialogMutation,
+    undefined,
+    {
+      successMessage: t_i18n('Saved filter successfully created'),
+    },
+  );
+
+  const handleChange = (event: ChangeEvent<HTMLInputElement>) => {
+    if (event.target.value === '') setFilterName(undefined);
+    else setFilterName(event.target.value);
+  };
+  const handleSubmitSaveFilter = () => {
+    commit({
+      variables: {
+        input: {
+          name: filterName,
+          filters: JSON.stringify(filters),
+          scope: localStorageKey,
+        },
+      },
+      updater: (store) => {
+        const scopeFilter = getSavedFilterScopeFilter(localStorageKey);
+        insertNode(store, 'SavedFilters__savedFilters', { filters: scopeFilter }, 'savedFilterAdd');
```

Comment:
```suggestion
        insertNode(store, 'SavedFilters_savedFilters', { filters: scopeFilter }, 'savedFilterAdd');
```
usually only one '_' for the insertNode key